includes/admin/class-sensei-learner-management.php

```diff
@@ -524,12 +524,9 @@ public function edit_date_started() {
 		if ( false === $date ) {
 			exit( '' );
 		}
-		$mysql_date = date( 'Y-m-d H:i:s', $date->getTimestamp() );
-		if ( false === $mysql_date ) {
-			exit( '' );
-		}
 
-		$updated = (bool) update_comment_meta( $comment_id, 'start', $mysql_date, $date_started );
+		$mysql_date = gmdate( 'Y-m-d H:i:s', $date->getTimestamp() );
```

Comment:
I think the name of the variable could be confusing:
for me, it implies it uses TZ from MySQL, which is not true.

I know it was here before, but maybe we can fix it now? :)